gammapy/spectrum/tests/test_flux_point_estimator.py

```diff
@@ -139,14 +145,20 @@ def test_run_ecpl(fpe_ecpl):
     @requires_dependency("iminuit")
     @requires_data("gammapy-data")
     def test_run_map_pwl(fpe_map_pwl):
-        fp = fpe_map_pwl.run(steps=["err", "norm-scan", "ts"])
+        fp = fpe_map_pwl.run()
 
         actual = fp.table["norm"].data
         assert_allclose(actual, [0.97922, 0.94081, 1.074426], rtol=1e-3)
 
         actual = fp.table["norm_err"].data
         assert_allclose(actual, [0.069966, 0.052617, 0.092854], rtol=1e-2)
 
+        actual = fp.table["counts"].data
+        assert_allclose(actual, [[44445, 0], [1911, 0], [292, 0]], rtol=1e-2)
```

Comment:
Should we need a tolerance for counts here?

Reply:
Done